src/libraries/System.Formats.Tar/src/System/Formats/Tar/TarHeader.Read.cs

```diff
@@ -28,10 +28,7 @@ internal sealed partial class TarHeader
             archiveStream.ReadExactly(buffer);
 
             TarHeader? header = TryReadAttributes(initialFormat, buffer);
-            if (header != null)
-            {
-                header.ProcessDataBlock(archiveStream, copyData);
-            }
+            header?.ProcessDataBlock(archiveStream, copyData);
```

Comment:
This should be part of the code style changes, right?

Reply:
Yes, it should. I'll revert this change.